elements/bulbs-carousel/bulbs-carousel.js

```diff
@@ -0,0 +1,174 @@
+import invariant from 'invariant';
+import { BulbsHTMLElement, registerElement } from 'bulbs-elements/register';
+import { moveChildren } from 'bulbs-elements/util';
+
+import './bulbs-carousel.scss';
+
+export const supportsCalcInTransform = (() => {
+  let div = document.createElement('div');
+  div.style.transform = 'translateX(calc(100% - 10px))';
+  return div.style.transform !== '';
+})();
```

Comment:
Can this be moved to utilities? Looks like a function that would be useful elsewhere.